zconf-ng.h.in

```diff
@@ -50,20 +45,18 @@
  for small objects.
 */
 
-                        /* Type declarations */
+/* Type declarations */
 
 
-#if defined(WINDOWS) || defined(WIN32)
+#if defined(_WIN32)
```

Comment:
Borland C++ again...

Reply:
why are so fixated on boarland. Consider it dead: https://github.com/zlib-ng/zlib-ng/pull/566#pullrequestreview-397506156

Reply:
__declspec() should be used only with Visual C++, so the test is incorrect...

As mentioned in the discussion, eliminating Borland C specific code should be separated as own pull request.

Reply:
@mtl1979 Will you prepare a PR that removes Borland C stuff then?

Reply:
It might not be wise to remove all of the preprocessor macros Borland C uses, but remove the ones that only Borland C use and then recheck the code with Cygwin, MSYS and MinGW to see any side-effects that need to be addressed. I don't have Cygwin, MSYS or MinGW installed on my current development PC and my old hard disk has only MSYS2 and MinGW-w64.

Reply:
I can make a PR after this commit that removes only Borland C if nobody else will.

Reply:
Our own guidelines dictate that we should not take over and fix patches started by other people. I don't agree with that guideline, but I don't see any reason why it should not apply here.